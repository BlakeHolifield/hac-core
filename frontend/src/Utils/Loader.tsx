import * as React from 'react';
import { Bullseye, Spinner } from '@patternfly/react-core';

const Loader = () => (
  <Bullseye>
    <Spinner />
  </Bullseye>
);

export default Loader;
