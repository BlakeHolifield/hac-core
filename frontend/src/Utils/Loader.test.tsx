import * as React from 'react';
import Loader from './Loader';
import { render } from '@testing-library/react';

describe('Loader', () => {
  it('should render correctly', () => {
    const container = render(<Loader />);
    expect(container.baseElement).toMatchSnapshot();
  });
});
