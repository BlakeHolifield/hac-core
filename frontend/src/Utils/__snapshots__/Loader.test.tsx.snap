// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render correctly 1`] = `
<body>
  <div>
    <div
      class="pf-l-bullseye"
    >
      <span
        aria-label="Contents"
        aria-valuetext="Loading..."
        class="pf-c-spinner pf-m-xl"
        role="progressbar"
      >
        <span
          class="pf-c-spinner__clipper"
        />
        <span
          class="pf-c-spinner__lead-ball"
        />
        <span
          class="pf-c-spinner__tail-ball"
        />
      </span>
    </div>
  </div>
</body>
`;
